CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+## Making changes to CitGM
+#### There are a few basic requirements for making changes to CitGM
+
+* Create an issue prior to submitting a PR as this can help speed up the process
+* Include tests for your code wherever possible
+* Include any documentation or manual changes where necessary
+* Ensure that `npm test` passes before submitting the PR
+* Please squash each logical change into a single commit
+* Follow the nodejs commit guidelines shown [here](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#step-3-commit)
```

Comment:
As @MylesBorins said in https://github.com/nodejs/citgm/pull/314#discussion_r96458927, we should probably discuss whether we're going to follow these guidelines. Looking at [recent commits](https://github.com/nodejs/citgm/commits/master) I guess we have been getting closer to the nodejs core style in terms of submitting PRs, we just haven't been adding any of the metadata on landing. 

So technically asking people to follow the commit guidelines is fine, we're using 50/72 chars and a `subsystem:` (we're just not adding a `PR-URL:` or `Reviewed-by:` on landing).